tests/conftest.py

```diff
@@ -319,7 +315,7 @@ async def _inner(wid, args):
         for script_name, n_args in {
             'config': 3,
             'list': 2,
-            'graph': 3,
+            'graph': 2,
```

Comment:
This is the one-liner needed to fix the tests.

Same as in https://github.com/cylc/cylc-rose/pull/394/files